internal/operator/operator.go

```diff
@@ -837,6 +873,19 @@ func (r *minioGroupBindingReconciler) Reconcile(ctx context.Context, req reconci
 			return failure(err)
 		}
 
+		l.Info("get minio group description")
```

Comment:
Minio group bindings will not throw an error if the user is already a member of the group.

To support workflows, we add a membership check that raises an error if `.spec.migrate` is false.  Otherwise, we proceed with the insert/update operation.